### ISSUE TITLE
Add hint when pushing xml

### ISSUE THREAD

**Issue by digimer** [ClusterLabs/pcs#241]
When complaining is too old, offer hint to check --config.

Sorry for the drunk feature request. :)

**Comment by tomjelinek** [comment]
`pcs cluster cib-push` may exit with:
```
Error: unable to push cib
Call cib_replace failed (-205): Update was older than existing configuration
```
This happens when the CIB being pushed is too old and therefore rejected by pacemaker.

Pcs should check for this specific reason of rejecting the CIB and provide a hint explaining what's going on and that it is possible to push the CIB with the `--config` option passed to pcs.

**Comment by tomjelinek** [comment]
This has been fixed in https://github.com/ClusterLabs/pcs/commit/e6b6289e1987dec34facfec0fb532f6640b45d2f and the fix is included in the latest 0.10.8 release.
